lib/govuk_navigation_helpers/taxonomy_sidebar.rb

```diff
@@ -19,8 +22,20 @@ def taxons
           title: parent_taxon.title,
           url: parent_taxon.base_path,
           description: parent_taxon.description,
+          related_content: content_related_to(parent_taxon),
```

Comment:
Let's also add comments here so people are aware this is a temporary fix that is meant to be migrated to the content store?